habits_project/habits/tasks.py

```diff
@@ -0,0 +1,14 @@
+from celery import shared_task
+from django.utils import timezone
+from .models import Habit
+from .services import send_telegram_message
+
+@shared_task
+def send_reminders():
+    now = timezone.localtime()
+    due = Habit.objects.filter(time__hour=now.hour, time__minute=now.minute)
```

Comment:
Так оптимальнее и ниже не придется через `getattr` получать данные
```suggestion
    due = Habit.objects.filter(time__hour=now.hour, time__minute=now.minute)
    due = due.select_related('user').exclude(user__telegram_chat_id__isnull=True)
```